src/wokwi_client/client.py

```diff
@@ -222,6 +251,10 @@ async def listen_pin(self, part: str, pin: str, listen: bool = True) -> Response
         """
         return await pin_listen(self._transport, part=part, pin=pin, listen=listen)
 
+    async def gpio_list(self) -> ResponseMessage:
```

Comment:
Let's return a useful type (ResponseMessage isn't really useful for someone who is using the API. 
IMHO the client should check for error, and throw if there's one, otherwise just return the result with a proper type (so users know which fields to expect and what types they have)